receiver/postgresqlreceiver/scraper_test.go

```diff
@@ -92,6 +92,44 @@ func TestScraperNoDatabaseMultiple(t *testing.T) {
 	require.NoError(t, scrapertest.CompareMetrics(expectedMetrics, actualMetrics))
 }
 
+func TestScraperWithResourceAttributeFeatureGate(t *testing.T) {
```

Comment:
Can we add a new case to `integration_test.go` as well?

Reply:
Yeah sure I can look into it. Looked into it initially and wasn't sure how to do it with the featuregate Registry while not affecting the other integration test. Do you happen to know of an example or if it could be solved using  the `	featuregate.GetRegistry().MustApply()` function?

Reply:
Testing around and I think the `featuregate.GetRegistry().MustApply()` should work 👍 probably should make sure that it cleans up the featuregate registry before that portion of the integration test completes.

Reply:
Just pushed an attempt at making an integration test let me know if that works!

Reply:
Thanks @schmikei. I think that looks like it'll work.